CMakeModules/FindMPI.cmake

```diff
@@ -273,8 +273,12 @@ if (DEFINED MPI_HOME)
 else()
     # Allow all paths, and add an extra path if set
     set(PATHOPT )
-    # Start with MPI_HOME from the environment, of given
-    set(_MPI_PREFIX_PATH $ENV{MPI_HOME})
+    # CPB 11/11/16 Don't look at MPI_HOME environment variable. Some MPI distros set this but others don't. This can mean that
+    # the MPI_HOME environment variable might clash with what has been specified via the -DMPI= option. The common MPI_HOME's
+    # are covered below regardless.
+    # Start with MPI_HOME from the environment, of given    
+    #set(_MPI_PREFIX_PATH $ENV{MPI_HOME})
+    set(_MPI_PREFIX_PATH )
```

Comment:
Don't we want this to be if NOT DEFINED MPI then set prefix path according to environment variable MPI_HOME otherwise don't set it